app/src/main/AndroidManifest.xml

```diff
@@ -5,87 +5,69 @@
+        android:usesCleartextTraffic="true"
         tools:targetApi="31">
+        <activity
+            android:name=".SettingsPage"
+            android:exported="true" />
         <activity
             android:name=".AboutUsPageActivity"
-            android:exported="false" >
-        </activity>
+            android:exported="false" />
         <activity
             android:name=".SuccessPage"
             android:exported="false" />
         <activity
             android:name=".ContactForm"
-            android:exported="true"></activity>
-
-        
+            android:exported="true" />
         <activity
             android:name=".Login"
             android:exported="true"
             android:theme="@style/Theme.Design.Light.NoActionBar">
             <intent-filter>
                 <action android:name="android.intent.action.MAIN" />
+
                 <category android:name="android.intent.category.LAUNCHER" />
             </intent-filter>
-            <!-- add intent filter back-->
+            <!-- add intent filter back -->
         </activity>
-
         <activity
             android:name=".SignUp"
             android:exported="true"
-            android:theme='@style/Theme.Design.Light.NoActionBar'></activity>
+            android:theme="@style/Theme.Design.Light.NoActionBar" />
         <activity
             android:name=".OutfitCreationActivity"
             android:exported="true"
-            android:theme="@style/Theme.Design.Light.NoActionBar"></activity>
-
+            android:theme="@style/Theme.Design.Light.NoActionBar" />
         <activity
-            android:name="com.example.mydigitalcloset.clothingUpload"
-            android:exported="false"></activity>
-
+            android:name=".clothingUpload"
+            android:exported="false" />
         <activity
-            android:name="com.example.mydigitalcloset.clothingFront"
-            android:exported="true"></activity>
-
-
-        <!-- duplicate login activity
+            android:name=".clothingFront"
+            android:exported="true" />
+        <!--
+ duplicate login activity
         <activity
             android:name=".ui.login.LoginActivity"
             android:exported="false"
             android:label="@string/title_activity_login" />
-
-          -->
+        -->
         <activity
             android:name=".closetPage.SavedOutfit"
             android:configChanges="orientation|keyboardHidden|screenSize"
-            android:exported="true">
-        </activity>
-
-        <activity
-            android:name=".clothingUpload"
-            android:exported="false">
-        </activity>
+            android:exported="true" />
 
         <activity
             android:name=".closetPage.AllSavedOufitsPage"
             android:configChanges="orientation|keyboardHidden|screenSize"
-            android:exported="true">
-       
-       </activity>
-
-        <activity
-            android:name=".clothingFront"
```

Comment:
make sure clothing front still has a section in manifest

Reply:
The clothing front still has a section in the manifest